### ISSUE TITLE
Helm chart available?

### ISSUE THREAD

**Issue by kir4h** [caicloud/event_exporter#39]
Sorry for not following the template, but it's rather a question than a feature request: Is there an existing Helm Chart available for `event_exporter`? Asking because comments in https://github.com/caicloud/event_exporter/blob/master/deploy/deploy.yaml suggest the usage of helm template, but I am unable to find the source.

I see also the included clusterrolebinding is bound to the general purpose clusterrole `view`. Is there a list of required privileges needed so that a clusterrole can be created for that purpose?

Thanks!

**Comment by lichuan0620** [comment]
I'll take this issue as a feature request with two separate features:

1. Helm chart for deployment
2. A more precise `ClusterRoleBinding`

/kind feature

**Comment by lichuan0620** [comment]
These features are nice to have after #37 is implemented

**Comment by kir4h** [comment]
> I'll take this issue as a feature request with two separate features:
> 
>     1. Helm chart for deployment
> 
>     2. A more precise `ClusterRoleBinding`
> 
> 
> /kind feature

Great! Once #37 is implemented I can help with the chart if needed

**Comment by danielmotaleite** [comment]
i found one in here: https://github.com/Facets-cloud/k8s-event-exporter-helm

EDIT: this one is probably even better
https://github.com/Facets-cloud/event_exporter/tree/master/helm
and their image have some improvements, namely  https://github.com/caicloud/event_exporter/pull/59 applied:
https://hub.docker.com/r/facetscloud/event-exporter/tags
